packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -1278,22 +1278,44 @@ export function createLodestarMetrics(
         name: "lodestar_shuffling_cache_size",
         help: "Shuffling cache size",
       }),
-      processStateInsertNew: register.gauge({
-        name: "lodestar_shuffling_cache_process_state_insert_new_total",
-        help: "Total number of times processState is called resulting a new shuffling",
-      }),
-      processStateUpdatePromise: register.gauge({
-        name: "lodestar_shuffling_cache_process_state_update_promise_total",
-        help: "Total number of times processState is called resulting a promise being updated with shuffling",
-      }),
-      processStateNoOp: register.gauge({
-        name: "lodestar_shuffling_cache_process_state_no_op_total",
-        help: "Total number of times processState is called resulting no changes",
-      }),
       insertPromiseCount: register.gauge({
         name: "lodestar_shuffling_cache_insert_promise_count",
         help: "Total number of times insertPromise is called",
       }),
+      hit: register.gauge({
+        name: "lodestar_shuffling_cache_hit_count",
+        help: "Count of shuffling cache hit",
+      }),
+      miss: register.gauge({
+        name: "lodestar_shuffling_cache_miss_count",
+        help: "Count of shuffling cache miss",
+      }),
+      shufflingBuiltMultipleTimes: register.gauge({
+        name: "lodestar_shuffling_cache_recalculated_shuffling_count",
+        help: "Count of shuffling that were build multiple times",
+      }),
+      shufflingPromiseNotResolvedAndThrownAway: register.gauge({
+        name: "lodestar_shuffling_cache_promise_not_resolved_and_thrown_away_count",
+        help: "Count of shuffling cache promises that were discarded and the shuffling was built synchronously",
+      }),
+      shufflingPromiseNotResolved: register.gauge({
+        name: "lodestar_shuffling_cache_promise_not_resolved_count",
+        help: "Count of shuffling cache promises that were requested before the promise was resolved",
+      }),
+      nextShufflingNotOnEpochCache: register.gauge({
+        name: "lodestar_shuffling_cache_next_shuffling_not_on_epoch_cache",
+        help: "The next shuffling was not on the epoch cache before the epoch transition",
+      }),
+      shufflingPromiseResolutionTime: register.histogram({
+        name: "lodestar_shuffling_cache_promise_resolution_time",
+        help: "Time from promise insertion until promise resolution when shuffling was ready",
+        buckets: [1, 10, 100, 1000],
```

Comment:
this value is not realistic, if it takes >= 2s would be bad hence I propose

```suggestion
        buckets: [0.5, 1, 1.5, 2],
```

Reply:
good catch!